src/components/link.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
Perhaps this file should be called `ExternalLink` since it opens it in a new tab